packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx

```diff
@@ -22,6 +27,30 @@ function isPartialHttpRequest(maybeHttpRequest: unknown): maybeHttpRequest is Pa
   );
 }
 
+interface ISchemaAndDescriptionProps {
+  schema: JSONSchema;
+  title?: string;
+}
+
+const SchemaAndDescription = ({ title: titleProp, schema }: ISchemaAndDescriptionProps) => {
+  const resolveRef = useInlineRefResolver();
+  const title = titleProp ?? schema.title;
+  return (
+    <Box py={2}>
+      {title && (
+        <Flex className="sl-flex sl-items-center flex-1" p={2}>
```

Comment:
`sl-flex-1` might be better after ui-kit is out and `sl-flex` is unnecessary.
Or even better `<Flex flex={1} alignItems="center" p={2}>`

Reply:
Actually there should be need for neither, as I got rid of the outer flexbox. Nice catch, I updated it.